### PR TITLE
Add Team markup

### PR DESCRIPTION
**Pull request by dpordomingo**
only the markup

**notice**
`partials/project/hero.html` was moved to `partials/hero.html` because it can be used in more pages (projects, careers, team and stack)

### CODE REVIEW

File: hugo/data/team.yml [hunk 1]
```diff
@@ -57,6 +59,14 @@ teams:
             name: Egor Bulychev
             position: Senior Software Engineer
             github: https://github.com/EgorBu
+          - picture: placeholder/new-man.png
+            name: __NEW_GUY__
```
Comment (erizocosmico) [inline hugo/data/team.yml]: leftover?

Comment (dpordomingo) [inline hugo/data/team.yml]: I have no data about the new guys...
I'll dig a bit more

**Comment by dpordomingo** [comment]
I added the info I could gather about Manu, Timofei and Konstantin

**Comment by dpordomingo** [comment]
@erizocosmico can/should u merge it?
or should I do it on my own?

**Pull request merged**